include/deal.II/numerics/matrix_creator.templates.h

```diff
@@ -622,7 +620,7 @@ namespace MatrixCreator
     {
       struct Scratch
       {
-        Scratch() = default;
+        Scratch() {}
```

Comment:
I think you are undoing a recent change here. Did you copy a modified version of this file over a recently updated one in your local git directory?

Reply:
I might have done this, as I don't remember changing these lines on my own